src/httprpc.cpp

```diff
@@ -93,10 +93,8 @@ static bool multiUserAuthorized(std::string strUserPass)
     std::string strUser = strUserPass.substr(0, strUserPass.find(":"));
     std::string strPass = strUserPass.substr(strUserPass.find(":") + 1);
 
-    if (gArgs.IsArgSet("-rpcauth")) {
-        //Search for multi-user login/pass "rpcauth" from config
-        BOOST_FOREACH(std::string strRPCAuth, gArgs.GetArgs("-rpcauth"))
-        {
+    //Search for multi-user login/pass "rpcauth" from config
+    BOOST_FOREACH(std::string strRPCAuth, gArgs.GetArgs("-rpcauth")) {
```

Comment:
Yeah, I was leaving it for #10193, but happy to do it here in the touched lines.
Specially since I'm not able to fix my problem with the reverse loop there :(